packages/db/src/models/base.ts

```diff
@@ -17,11 +17,19 @@ class BasePgModel<ItemType, RecordType extends { cumulus_id: number }> {
     this.tableName = tableName;
   }
 
-  get(
+  async get(
     knexOrTransaction: Knex | Knex.Transaction,
     params: Partial<RecordType>
   ) {
```

Comment:
Nit (and not your PR), we should consider being explicit re: this method's return

Reply:
(and/or the other methods in this file)

Reply:
Yup! Easy enough. I learned some thing about Knex's handling of types too 😄 

bcde23b7748bcf0896d337701e8fd7b7c583fbf2

Reply:
👍🏻     I'm fine with punting the other method changes, we can add that as a seperate PR.   Thanks!